src/remediations/fifi.js

```diff
@@ -41,17 +49,49 @@ exports.getSatelliteId = function (facts) {
     return null;
 };
 
+function defineRHCEnabledExecutor (satellites) {
+    const lastElement = _.last(satellites);
+    if (lastElement.id === null) {
+        const nonRhcSystems = _.filter(lastElement.systems, system => _.isUndefined(system.rhc_client));
+
+        lastElement.systems = _.filter(lastElement.systems, system => !_.isUndefined(system.rhc_client));
+
+        satellites.push({id: null, systems: nonRhcSystems});
+    }
+}
+
+function fetchRHCStatus (systems) {
+    let rhcConnectionStatus = 'disconnected';
```

Comment:
Have you considered checking the connection status with cloud-connector similarly to how we do with receptor? The fact that a host has `rhc_client` defined does not necessarily indicate that it is connected right now. No need to make it part of this initial request just something to consider.

Reply:
Will do!  I was planning on adding in the check to do that after this initial request.